### DISCUSSION TITLE
push notification when PR was reviewed

### DISCUSSION THREAD

**Discussion by sawaca96** [community/community#3672] [Mobile] [answered]
I want to get notification when my PR was reviewed by someone
and also get notification any comment about assigned PR  OR any comment about PR which I am reviewer

**Comment by brianlovin** [comment, marked as answer]
Thanks for the notes here. We just rolled out some new push notification types last week (available in Profile → Settings → Notifications). The new types won't cover your exact needs, but hopefully get closer to what you're looking for in terms of knowing when people need your attention on specific threads.

We probably won't support arbitrary push notifications for _any event type_ in a thread where you're a participant or a reviewer, as that could get really noisy. But: we will keep adding new push notification types in the future as we learn more!
